spyder/utils/encoding.py

```diff
@@ -224,16 +225,23 @@ def to_unicode(string):
     return string
     
 
-def write(text, filename, encoding='utf-8', mode='wb'):
+def write(text, filename, encoding='utf-8', mode='wb', overwrite=True):

```

Comment:
I don't think we need to set `overwrite` to any other than `True`, so please remove this keyword arg from here.